doc/uftrace-record.md

```diff
@@ -21,12 +21,87 @@ This data can then be inspected later on, using `uftrace replay` or
 `uftrace report`.
 
 
-OPTIONS
-=======
--b *SIZE*, \--buffer=*SIZE*
-:   Size of internal buffer in which trace data will be saved.  Default size is
-    128k.
+RECORD OPTIONS
+==============
+-A *SPEC*, \--argument=*SPEC*
+:   Record function arguments.  This option can be used more than once.
+    See *ARGUMENTS*.
+
+-R *SPEC*, \--retval=*SPEC*
+:   Record function return values.  This option can be used more than once.
+    See *ARGUMENTS*.
+
+-P *FUNC*, \--patch=*FUNC*
+:   Patch FUNC dynamically.  This is only applicable binaries built by
+    gcc with `-pg -mfentry -mnop-mcount` or clang with `-fxray-instrument`.
+    This option can be used more than once.  See *DYNAMIC TRACING*.
+
+-E *EVENT*, \--event=*EVENT*
+:   Enable event tracing.  The event should be available on the system.
+
+-S *SCRIPT_PATH*, \--script=*SCRIPT_PATH*
+:   Run a given script to do additional work at the entry and exit of function
+    during target program execution.
+    The type of script is detected by the postfix such as '.py' for python.
+    See *SCRIPT EXECUTION*.
+
+-W, \--watch=*POINT*
+:   Add watch point to display POINT if the value is changed.  See *WATCH POINT*.
+
+-a, \--auto-args
+:   Automatically record arguments and return values of known functions.
+    These are usually functions in standard (C language or system) libraries
+    but if debug info is available it includes functions in the user program.
+
+-l, \--nest-libcall
+:   Trace function calls between libraries.  By default, uftrace only record
+    library call from the main executable.  Implies `--force`.
+
+-k, \--kernel
+:   Trace kernel functions as well as user functions.  Only kernel entry/exit
+    functions will be traced by default.  Use the `--kernel-depth` option to
+    override this.
+
+-K *DEPTH*, \--kernel-depth=*DEPTH*
+:   Set kernel max function depth separately.  Implies `--kernel`.
 
+-H *HOST*, \--host=*HOST*
+:   Send trace data to given host via the network, not writing to files.
+    The `uftrace recv` command should be run on the destination host to receive
+    the data.
+
+\--port=*PORT*
+:   When sending data to the network (with `-H`), use the given port instead of
+    the default (8090).
```

Comment:
I'm not sure yet, but I think `--host 127.0.0.1` and `--port 8888` options can be easily used as `--host 127.0.0.1:8888`. We can consider it later.

Reply:
Yes, that should be possible too